src/medCore/database/medAbstractDatabaseImporter.cpp

```diff
@@ -202,89 +202,91 @@ void medAbstractDatabaseImporter::importFile ( void )
         currentFileNumber++;
 
         QFileInfo fileInfo ( file );
+        if (fileInfo.size() != 0)
```

Comment:
There are 2 kinds of error handling in this function:
 * stop the current reading of the file and switch to the following file (`continue;`)
 * stop the reading of the batch of files (`return`)

In this case, i do a `continue` (and i can add a message). What do you prefer if a file is empty? We stop everything or just this file?